django/db/backends/sqlite3/features.py

```diff
@@ -147,6 +147,16 @@ def introspected_field_types(self):
             "SmallAutoField": "AutoField",
         }
 
+    @property
+    def max_query_params(self):
+        """
+        SQLite has a variable limit per query. The limit can be changed using
+        the SQLITE_MAX_VARIABLE_NUMBER compile-time option (which defaults to
+        999 in versions < 3.32.0 or 32766 in newer versions) or lowered per
+        connection at run-time with setlimit(SQLITE_LIMIT_VARIABLE_NUMBER, N).
+        """
```

Comment:
I moved and updated the docstring of `bulk_batch_size` to here as it's more relevant